lib/src/realm_object.dart

```diff
@@ -16,37 +16,107 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-// ignore_for_file: native_function_body_in_non_sdk_code
+import 'native/realm_core.dart';
+import 'realm_property.dart';
 
-/**
- *  The file is intentionaly not follwoing the dart naming guidelines. The name is used from native code by convention
- */
+abstract class RealmAccessor {
+  T get<T extends Object>(RealmObject object, String name);
+  void set<T extends Object>(String name, T value);
+}
 
-import 'list.dart';
-import 'realm_property.dart';
+class RealmValuesAccessor implements RealmAccessor {
+  final Map<String, Object> _values = <String, Object>{};
```

Comment:
Shouldn't this be `Map<String, Object?>` to allow for null values?

Reply:
good point will fix